app/UiTPAS/Listeners/GiveUitpasPermissionsToTestOrganizer.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\UiTPAS\Listeners;
+
+use App\Domain\Integrations\Environment;
+use App\Domain\Integrations\Events\IntegrationCreated;
+use App\Domain\Integrations\Events\IntegrationUpdated;
+use App\Domain\Integrations\IntegrationType;
+use App\Domain\Integrations\Repositories\IntegrationRepository;
+use App\Keycloak\Realm;
+use App\UiTPAS\UiTPASApiInterface;
+use App\UiTPAS\UiTPASConfig;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+final class GiveUitpasPermissionsToTestOrganizer implements ShouldQueue
```

Comment:
Does this class name really describe what happens? Is the keycloak client ID of the integration not added to the test organizer?

Reply:
It think it is `AddUiTPASPermissionsToOrganizerForIntegration`